### ISSUE TITLE
[markdown rendering issue] stick words with italic is not working

### ISSUE THREAD

**Issue by Canine89** [javascript-tutorial/en.javascript.info#2040]
Hi iliakan.
I'm participating in the javascript.ko project.
By the way, I found markdown rendering issue.
It may only occur in Korean documents.
See below screenshot.

![image](https://user-images.githubusercontent.com/16553217/89148111-d7768900-d593-11ea-8c64-750fd20eb7a0.png)
![image](https://user-images.githubusercontent.com/16553217/89148117-dba2a680-d593-11ea-92f0-b164e8d67f07.png)

First image is the problem.
Korean sometimes has to stick words together.
I Hope this helps fix the issue.

**Comment by iliakan** [comment]
Right now italic requires spaces to the both sides of `*`. So in expressions like `5*2` the star is not considered a special markup character.

I can tweak this rule, but tell me how? We don't want the star `*` to be mistreated in other situations.

Also, is this a real problem? Can you rephrase Korean?

@Violet-Bora-Lee what you think?

**Comment by Violet-Bora-Lee** [comment]
Until now, I've added a whitespace in order to avoid this problem.
It is wrong spell inserting whitespace actually.

Is it easy to implement? If not, I could add some guide on the README file.

**Comment by iliakan** [comment]
Hi,

The question is deeper than one might thing.

Some time ago we rewrote our parser to base it on <https://github.com/markdown-it/markdown-it>, that implements the [CommonMark](https://commonmark.org/) specification.

The italic/bold thing is handled by that parser, according to the spec.

I decided to see what the CommonMark spec says about that case. 

At <https://commonmark.org/help/tutorial/02-emphasis.html> I entered `*마크다운(script)*렌더링` (copied-pasted arbitrary Korean chars, otherwise I have no Korean on my keyboard).

And it remained `*마크다운(script)*렌더링` (as is), the CommonMark didn't convert it.

So if we need to fix the problem, we need to tell the guys who make the CommonMark specification about it. Then they can hopefully update the spec, and then the parser updates too, so everyone's happy.

I suggest going to https://talk.commonmark.org/ and making a topic there, such as "BUG in Korean" and then describe the issue. Maybe there's a way out.

P.S. Please note: the problem occurs only if I put "(script)" in there. Maybe `)` has something to do with it.
